api/topic/topic.go

```diff
@@ -323,7 +347,13 @@ func InitPrivateTopic() {
 }
 
 // Insert creates a new topic. User is read write on topic
-func Insert(topic *tat.Topic, user *tat.User) error {
+func Insert(topic *tat.Topic, u *tat.User) error {
+	keys, _ := cache.Client().SMembers(cache.Key("tat", "users", u.Username, "topics")).Result()
+	if len(keys) > 0 {
+		cache.Client().Del(keys...)
+	}
+	cache.Client().Del(cache.Key("tat", "users", u.Username, "topics"))
```

Comment:
we have to delete all keys tat:users:<username>:topics for all users. This key is is probably out of date for some users for this new topic.